src/org/spdx/licenselistpublisher/licensegenerator/LicenseJsonFormatWriter.java

```diff
@@ -110,7 +104,7 @@ public void setJsonFolderDetails(File jsonFolderDetails) {
 
 	@Override
 	public void writeLicense(SpdxListedLicense license, boolean deprecated, String deprecatedVersion) throws IOException {
-		license.setCrossRef(UrlHelper.buildUrlDetails(license.getSeeAlso()));
+		license.setCrossRef(CrossRefHelper.buildUrlDetails(license.getSeeAlso()));
```

Comment:
These would be removed since it is built in the builder not in the writers per my previous comment